src/Microsoft.Diagnostics.ExtensionCommands/EEHeap.cs

```diff
@@ -0,0 +1,506 @@
+﻿using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Runtime;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    [Command(Name = "eeheap", Aliases = new string[] { "EEHeap" }, Help = "Displays information about native memory that CLR has allocated.")]
```

Comment:
The mixed case alias isn't really needed.  The code in strike.cpp will make sure !EEHeap works until windbg and for lldb and dotnet-dump analyze all the commands are lower case.

Reply:
Fixed.  For real extension commands (ones without sos.dll exports), does case matter here?  For example, does `!ext MADDRESS` work the same as `!ext maddress` or do I need to define aliases?  (Either behavior is fine for me, but I'm curious about the behavior and how I should define commands.)

Reply:
All the C# extensions are case sensitive so only !ext maddress will work.  It has been suggested that we make them case insensitive which I'm still thinking about and the implementation my be a little difficult because we use System.CommandLine for our parsing and it is case sensitive.

Reply:
FYI, !ext is replaced by !sos because "sos \<command\>" (!sos under windbg) can be used consistently in all the hosts: lldb, dotnet-dump and VS. It is still there in windbg SOS for backwards compatible.